### PR TITLE
Invalid locale fixed ku_IQ -> kmr_KU

### PR DESCRIPTION
**Pull request by AramRafeq**
[[中文版模板 / Chinese template](https://github.com/ant-design/ant-design/blob/master/.github/PULL_REQUEST_TEMPLATE/pr_cn.md)]

### 🤔 This is a ...

- [ ] New feature
- [x] Bug fix
- [x] Site / documentation update
- [ ] Demo update
- [ ] Component style update
- [ ] TypeScript definition update
- [ ] Bundle size optimization
- [ ] Performance optimization
- [ ] Refactoring
- [ ] Code style optimization
- [ ] Test Case
- [ ] Branch merge
- [ ] Other (about what?)

### 📝 Changelog

| Language   | Changelog |
| ---------- | --------- |
| 🇺🇸 English |   this is a fix for [this issue](https://github.com/ant-design/ant-design/issues/25778)        |
| 🇨🇳 Chinese | 这是一个修复 [这个问题](https://github.com/ant-design/ant-design/issues/25778)   |

### ☑️ Self Check before Merge

⚠️ Please check all items below before review. ⚠️

- [ ] Doc is updated/provided or not needed
- [ ] Demo is updated/provided or not needed
- [ ] TypeScript definition is updated/provided or not needed
- [ ] Changelog is provided or not needed

-----
[View rendered docs/react/i18n.en-US.md](https://github.com/AramRafeq/ant-design/blob/master/docs/react/i18n.en-US.md)
[View rendered docs/react/i18n.zh-CN.md](https://github.com/AramRafeq/ant-design/blob/master/docs/react/i18n.zh-CN.md)

### CODE REVIEW

**Comment by AramRafeq** [comment]
## Note
this is a breaking change PR was created for each of 
* **rc-paginate** [check PR](https://github.com/react-component/pagination/pull/291)
* **rc-picker** [check PR](https://github.com/react-component/picker/pull/110)

File: components/locale/kmr_KU.tsx [hunk 1]
```diff
@@ -1,11 +1,11 @@
...
-import Calendar from '../calendar/locale/ku_IQ';
+import Pagination from 'rc-pagination/lib/locale/kmr_KU';
+import DatePicker from '../date-picker/locale/kmr_KU';
+import TimePicker from '../time-picker/locale/kmr_KU';
+import Calendar from '../calendar/locale/kmr_KU';
 import { Locale } from '../locale-provider';
 
 const localeValues: Locale = {
-  locale: 'ku-iq',
+  locale: 'kmr-ku',
```
Comment (afc163) [inline components/locale/kmr_KU.tsx:8]: It would be a breaking change. We should keep compatible to original usage.

Comment (AramRafeq) [reply components/locale/kmr_KU.tsx:8]: yes this PR could remain open untill 
**rc-picker** and **rc-paginate** gets updated i have created PR on those two packages also, after they update we can change dependency for those packages to latest change

Comment (yoyo837) [reply components/locale/kmr_KU.tsx:8]: I understand what `afc163` means, keep `ku_IQ` and add new `kmr_KU`, is that right?

Comment (AramRafeq) [reply components/locale/kmr_KU.tsx:8]: @yoyo837 this is an alternative you could keep ku_IQ and make an alias of it named kmr_KU this is a solution also but it should be deprecated in the future

**Pull request closed without merging**